Content.Server/_NF/Trade/TradeCrateComponent.cs

```diff
@@ -0,0 +1,56 @@
+using System.Threading;
+using Content.Shared._NF.Trade;
+
+namespace Content.Server._NF.Trade;
+
+/// <summary>
+/// This is used to mark an entity to be used as a trade crate
+/// </summary>
+[RegisterComponent]
+public sealed partial class TradeCrateComponent : SharedTradeCrateComponent
+{
+    /// <summary>
+    /// The value of the crate, in spesos, when at its destination.
+    /// </summary>
+    [DataField]
+    public int ValueAtDestination;
+
+    /// <summary>
+    /// The value of the crate, in spesos, when delivered elsewhere.
+    /// </summary>
+    [DataField]
+    public int ValueElsewhere;
+
+    /// <summary>
+    /// If non-zero, this crate will be an express delivery.
+    /// </summary>
+    [DataField]
+    public TimeSpan ExpressDeliveryDuration = TimeSpan.Zero;
+
+    /// <summary>
+    /// If non-null, the package must be redeemed before this time to arrive unpenalized.
+    /// </summary>
+    [ViewVariables]
```

Comment:
datafield + timeoffsetserializer + autopausedfield

Reply:
DataField: no, it makes no sense to set this in the prototype.
Everything else, sure.